iconv on Alpine does not support conversion to ASCII//TRANSLIT

--- a/apps/user_ldap/lib/Access.php
+++ b/apps/user_ldap/lib/Access.php
@@ -1200,7 +1200,7 @@
 
 		// Transliteration
 		// latin characters to ASCII
-		$name = iconv('UTF-8', 'ASCII//TRANSLIT', $name);
+		$name = iconv('UTF-8', 'ASCII', $name);
 
 		// Replacements
 		$name = str_replace(' ', '_', $name);
